examples/model_compress/lottery_torch_mnist_fc.py

```diff
@@ -76,9 +102,12 @@ def test(model, test_loader, criterion):
         pruner.prune_iteration_start()
         loss = 0
         accuracy = 0
-        for epoch in range(10):
+        for epoch in range(args.train_epochs):
             loss = train(model, train_loader, optimizer, criterion)
             accuracy = test(model, test_loader, criterion)
             print('current epoch: {0}, loss: {1}, accuracy: {2}'.format(epoch, loss, accuracy))
         print('prune iteration: {0}, loss: {1}, accuracy: {2}'.format(i, loss, accuracy))
-    pruner.export_model('model.pth', 'mask.pth')
+
+    # reset weight to original untrained model to export winning ticket
+    pruner.load_model_state_dict(pruner._model_state)
+    pruner.export_model('model_winning_ticket.pth', 'mask_winning_ticket.pth')
```

Comment:
so, the exported one might not be a winning ticket?

Reply:
Thanks, updated to export winning ticket.